README.md

```diff
@@ -64,21 +71,41 @@ zip -r -q compiled_all_sql.zip target/compiled_all_sql
 
 ### Integration with Report Portal
 
-https://reportportal.io/ helps you to manage your test launches. Here at EPAM we're using this tool to manage over 4,000 DBT tests  
+https://reportportal.io/ helps you to manage your test launches. Here at EPAM we're using this tool to manage over 4,000 DBT tests
 
 In order to upload your test run to reportportal you can use the following script:
 ```shell
 dbt-junitxml parse target/run_results.json target/manifest.json dbt_test_report.xml
 zip dbt_test_report.zip dbt_test_report.xml
 REPORT_PORTAL_TOKEN=`Your token for Report Portal`
-RESPONSE=`curl -X POST "https://reportportal.io/api/v1/epm-plxd/launch/import" -H  "accept: */*" -H  "Content-Type: multipart/form-data" -H  "Authorization: bearer ${REPORT_PORTAL_TOKEN}" -F "file=@dbt_test_report.zip;type=application/x-zip-compressed"`
+RESPONSE=`curl -X POST "https://reportportal.epam.com/api/v1/plugin/{project_name}/JUnit/import" -H  "accept: */*" -H  "Content-Type: multipart/form-data" -H  "Authorization: bearer ${REPORT_PORTAL_TOKEN}" -F "file=@dbt_test_report.zip;type=application/x-zip-compressed"`
 LAUNCH_ID=`echo "${RESPONSE}" | sed 's/.*Launch with id = \(.*\) is successfully imported.*/\1/'`
 ```
 
+### Test Case Attribute displayed in Report Portal
+
+Since 0.2.2 version you will be able to put attributes within an junit xml report. For this you'll need additionally provide --custom_properties:
```

Comment:
Let's mention that it's only useful for large dbt projects where we want to group / tag DQ tests in some groups according to the file structure

Reply:
fixed